awscli-plugin-granica/bolt_cli_hooks.py

```diff
@@ -34,10 +34,10 @@ def _activate(parsed_args, **kwargs):
         except Exception as e:
             pass
 
-    if 'bolt_custom_domain' in profile and region is not None:
+    if 'granica_custom_domain' in profile and region is not None:
```

Comment:
Change file names bolt_router.py and bolt_cli_hooks.py ?

Reply:
They are not visible to the Cx, trying to keep the scope narrow